Scripts/Python/xOptionsMenu.py

```diff
@@ -1403,7 +1433,12 @@ def OnTimer(self,id):
             if PtIsDemoMode():
                 gLiveMovie = ptMoviePlayer(kDemoMovieName,self.key)
             else:
-                gLiveMovie = ptMoviePlayer(kLiveMovieName,self.key)
+                gLiveMovieList = []
+                for file in os.listdir(kLiveMovieDir):
+                    if os.stat(f'{kLiveMovieDir}\{file}').st_size > 1000:
```

Comment:
Is there a specific reason for this particular choice in filesize?

Reply:
Moula has a file called "PreBranding.webm" that is 826 bytes, this was to prevent that from being shown and causing the Trailer GUI to exit immediately due to it closing when movies end.

Reply:
It might be a good idea to specifically exclude that video if that's the purpose rather than implicitly by some heuristic.

Filtering out dot files is probably a good idea to do as well.  I think a whitelist of supported video extensions might be the better route for that.

Reply:
I don't want to specifically exclude it because there may be a time when the PreBranding video actually has legit video to play, and it will then be larger than 1000.
Right now the code will also play the cyan worlds intro.

Yeah, I agree that this code needs some more work to only allow supported video extensions

Reply:
Okay after much struggle, ive got code that only allows webm files greater than 1000 bytes.
Tested with No files, Folder ending in ".webm", and a text file in the avi folder